guide/how-it-works/ecash/design-best-practices.md

```diff
@@ -0,0 +1,496 @@
+
+{% include tip/open.html color="blue" icon="info" label="Mint Information Needed" %}
+
+Keep a record of the mints you are connected to. During the restoration process, you will need to reconnect to these specific mints to re-verify your tokens. Without this information, you will not be able to fully restore your wallet.
+
+{% include tip/close.html %}
+
+### Restoration process
+{% include image-gallery.html pages = page.images_restore %}
+
+1. **Seed Phrase Entry**: When restoring a wallet, users enter their seed phrase into their wallet. This seed phrase regenerates secrets that may have been used to mint ecash.
+
+2. **Mint(s) Entry**: Users must input the mint URL for each mint they used before. This ensures their wallet can properly reconnect and recreate their tokens.
+
+3. **Verification**: The mint(s) checks these blinded secrets against its records to confirm if it has been seen before and can reissue the signature.
+
+{% include tip/open.html color="blue" icon="info" label="Privacy Considerations During Verification" %}
+
+Verifying tokens after restoration might temporarily compromise their privacy. The mint needs to revalidate the tokens, which can potentially expose the transaction history associated with those tokens. Be mindful of this aspect if privacy is a significant concern for your use case.
+
+{% include tip/close.html %}
+
+### Cashu metadata fields: explanation and usage
+
+[NUT06](https://github.com/cashubtc/nuts/blob/main/06.md) provides a standardized way to display metadata for Cashu mints. Incorporating these metadata fields makes your application more transparent and informative. Here's a breakdown of each field and how it can be used in wallet design:
+
+#### Mint Identity and Version
+
+<div class="center" markdown="1">
+
+{% include picture.html
+   image = "/assets/images/guide/how-it-works/ecash/cashu-id1.png"
+   retina = "/assets/images/guide/how-it-works/ecash/cashu-id1@2x.png"
+   modalImage = "/assets/images/guide/how-it-works/ecash/cashu-id1@2x.png"
+   alt-text = "Mint identity and version display in a wallet interface"
+   width = 250
+   height = 400
+   layout = "float-right-desktop -background -shadow"
+   caption = "Consider the most important information to display to the user about the mint."
+%}
+
+- `name`: The name of the mint. Display this prominently in mint listings and details pages to help users quickly identify different mints.
+- `pubkey`: The hex public key of the mint. While this may not be directly displayed to users, it's crucial for security verification. Consider showing a shortened version or QR code for advanced users.
```

Comment:
The pubkey actually isn't used for anything in production at the moment

Reply:
Noted! Removed from the guide for now.